### PR TITLE
add database project and create table migration script.

### LINKED ISSUE #2: Create Tables in DB

### PR DESCRIPTION
**Pull request by oahmed-OS**
close #2

### CODE REVIEW

**Comment by kyukibug** [comment]
ihjkbjjnkkm

File: ShoppingTogether.API.sln [hunk 1]
```diff
@@ -3,7 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.7.34018.315
 MinimumVisualStudioVersion = 10.0.40219.1
```
Comment (kyukibug) [inline ShoppingTogether.API.sln:5]: mk

**Review by kyukibug — APPROVED** [review]

File: ShoppingTogether.API.sln [hunk 2]
```diff
@@ -3,7 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
...
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ShoppingTogether.API", "ShoppingTogether.API\ShoppingTogether.API.csproj", "{04EA070A-870C-47A2-B5F0-4DE933FA72B1}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ShoppingTogether.API", "ShoppingTogether.API\ShoppingTogether.API.csproj", "{04EA070A-870C-47A2-B5F0-4DE933FA72B1}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ShoppingTogher.Database", "ShoppingTogher.Database\ShoppingTogher.Database.csproj", "{E32478E1-6A33-4B1D-B281-FA89866138EF}"
 EndProject
 Global
```
Comment (kyukibug) [inline ShoppingTogether.API.sln:10]: tyucvg huj bknm,l

**Review by kyukibug — APPROVED** [review]
i like it

**Review by kyukibug — APPROVED** [review]
good

**Comment by oahmed-OS** [comment]
close #2

**Pull request merged**